examples/market_maker/on_chain_market_maker.vy

```diff
@@ -7,21 +7,21 @@ totalTokenQty: public(uint256(currency_value))
 # Constant set in `initiate` that's used to calculate
 # the amount of ether/tokens that are exchanged
 invariant: public(uint256(wei * currency_value))
-token_address: address(ERC20)
+tokenAddress: address(ERC20)
 owner: public(address)
 
 # Sets the on chain market maker with its owner, intial token quantity,
 # and initial ether quantity
 @public
 @payable
-def initiate(token_addr: address, token_quantity: uint256(currency_value)):
+def initiate(tokenAddr: address, tokenQuantity: uint256(currency_value)):
```

Comment:
@Dan-Nolan @fubuloubu I think we should keep arguments and local variables as is, camelCase for those too are a bit too extreme in my opinion.

Reply:
Sounds good, I'll revert the latest commit